utils/queryApiGateway.ts

```diff
@@ -126,7 +126,7 @@ export function reponseIsNullish(apiBody: Claim): boolean {
   // This is a trick to make a deep-copy of a JSON object
   const response: Claim = JSON.parse(JSON.stringify(apiBody)) as Claim
```

Comment:
without _some missing piece_ to avoid the `as`, we can't go further in the Typescript refactor. But we have met the goal of the initial ticket!